README.md

```diff
@@ -168,86 +180,55 @@ project (typically as a git submodule), for example in `third_party/HighFive`.
+but wont search or link any optional dependencies such as Boost.
 
 ### Regular Installation of HighFive
 
-Alternatively you can install HighFive once and use it in several projects via
-`find_package()`. First one should clone the sources:
-```bash
-git clone --recursive https://github.com/BlueBrain/HighFive.git HighFive-src
-```
-By default CMake will install systemwide, which is likely not appropriate. The
-instruction below allow users to select a custom path where HighFive will be
-installed, e.g. `HIGHFIVE_INSTALL_PREFIX=${HOME}/third_party/HighFive` or some
-other location. The CMake invocations would be
-```bash
-cmake -DHIGHFIVE_EXAMPLES=Off \
-      -DHIGHFIVE_USE_BOOST=Off \
-      -DHIGHFIVE_UNIT_TESTS=Off \
-      -DCMAKE_INSTALL_PREFIX=${HIGHFIVE_INSTALL_PREFIX} \
-      -B HighFive-src/build \
-      HighFive-src
-
-cmake --build HighFive-src/build
-cmake --install HighFive-src/build
-```
-This will install (i.e. copy) the headers to
-`${HIGHFIVE_INSTALL_PREFIX}/include` and some CMake files into an appropriate
-subfolder of `${HIGHFIVE_INSTALL_PREFIX}`.
+Alternatively, HighFive can be install and "found" like regular software.
 
-The projects `CMakeLists.txt` should add the following:
+The project's `CMakeLists.txt` should add the following:
 ```cmake
-# ...
-add_executable(foo foo.cpp)
-
 find_package(HighFive REQUIRED)
 target_link_libraries(foo HighFive)
 ```
 
-**Note:** If HighFive hasn't been installed in a default location, CMake needs
-to be told where to find it which can be done by adding
-`-DCMAKE_PREFIX_PATH=${HIGHFIVE_INSTALL_PREFIX}` to the CMake command for
-building the project using HighFive. The variable `CMAKE_PREFIX_PATH` is a
-semi-colon `;` separated list of directories.
-
-**Note:** `find_package(HighFive)` will search and "link" HDF5 and optional
-dependencies such as Boost.
+**Note:** `find_package(HighFive)` will search for HDF5. "Linking" to
+`HighFive` includes linking with HDF5. The two commands will not search for or
+"link" to optional dependencies such as Boost.
 
-### The Bailout Approach
-Since both `add_subdirectory` and `find_package` will trigger finding HDF5 and
-other optional dependencies of HighFive as well as the `target_link_libraries`
-code for "linking" with the dependencies, things can go wrong.
+### Bailout Approach
 
-Fortunately, HighFive is a header only library and all that's needed is the
-headers. Preferably, the version obtained by installing HighFive, since those
-include `H5Version.hpp`. Let's assume they've been copied to
-`third_party/HighFive`. Then one could create a target:
+To prevent HighFive from searching or "linking" to HDF5 the project's
+`CMakeLists.txt` should contain the following:
 
-```bash
-add_library(HighFive INTERFACE)
-target_include_directory(HighFive INTERFACE ${CMAKE_CURRENT_SOURCE_DIR}/third_party/HighFive/include)
+```cmake
+set(HIGHFIVE_FIND_HDF5 Off)
+find_package(HighFive REQUIRED)
+# Alternatively:
+# add_subdirectory(third_party/HighFive)
```

Comment:
It might not be crystal clear that this is an alternative for `find_package`

Reply:
Thank you, we just spent the two immediately preceding subsections explaining the difference.